data/locus_coeruleus/locus_coeruleus.R

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env Rscript
+
+# Author_and_contribution: Niklas Mueller-Boetticher; created template
+# Author_and_contribution: Florian Heyl (@heylf); created code
+
+# H_E.json and H_E.tiff not public. Request for access is still unanswered.
+
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(SpatialExperiment))
+suppressPackageStartupMessages(library(SingleCellExperiment))
+suppressPackageStartupMessages(library(WeberDivechaLCdata))
+suppressPackageStartupMessages(library(Matrix))
+
+option_list <- list(
+  make_option(
+    c("-o", "--out_dir"),
+    type = "character", default = NULL,
+    help = "Output directory to write files to."
+  )
+)
+
+description <- "Load data (Visium) for Locus_coeruleus from Lukas M. Weber at al. (2022); 
+The gene expression landscape of the human locus coeruleus revealed 
+by single-nucleus and spatially-resolved transcriptomics"
+
+opt_parser <- OptionParser(
+  usage = description,
+  option_list = option_list
+)
+opt <- parse_args(opt_parser)
+
+out_dir <- opt$out_dir
+
+args = commandArgs(trailingOnly=TRUE)
+
+# Load data
+spe <- WeberDivechaLCdata_Visium()
+
+coords <- as.data.frame(spe@int_colData$spatialCoords)
+colnames(coords) <- c('x', 'y')
+coords_rownames <- rownames(spe@int_colData$spatialCoords)
+
+counts <- spe@assays@data$counts
+
+counts_func <- function(x){
+  fields <- unlist(strsplit(x, "_"))
+  if ( length(fields) == 4 ){
+    return(paste(fields[1:3], collapse = "_"))
+  } else {
+    return(paste(fields[1:4], collapse = "_"))
+  }
+}
+counts_lc <- unlist(lapply(colnames(counts), counts_func))
+
+LC_samples <- unique(spe@colData$sample_id)
+
+out_dir <- 'out_dir'
```

Comment:
One more thing. Any reason for this additional specific definition of `out_dir` instead of the `args$out_dir` you've defined before?

Reply:
Ach ups. Well its the special `I forgot to remove out_dir`. I fixed it

Reply:
haha thanks! It all happened to us xd